pkg/apis/sparkoperator.k8s.io/v1beta2/types.go

```diff
@@ -286,6 +286,9 @@ type BatchSchedulerConfiguration struct {
 	// PriorityClassName stands for the name of k8s PriorityClass resource, it's being used in Volcano batch scheduler.
 	// +optional
 	PriorityClassName *string `json:"priorityClassName,omitempty"`
+	// CustomRequestResource stands for the resource list custom request for, it's being used in Volcano batch scheduler.
+	// +optional
+	CustomRequestResource apiv1.ResourceList `json:"customRequestResource,omitempty"`
```

Comment:
Can we rename this to simply `resources`?

Reply:
Yes, we can use "minRes" or "minResource".

Reply:
done